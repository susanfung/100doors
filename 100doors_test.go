package main

import (
	"testing"
	approvals "github.com/approvals/go-approval-tests"
)

func TestInitialState(t *testing.T) {
	approvals.VerifyArray(t, Answer(0))
}

func TestFirstPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(1))
}

func TestSecondPass(t *testing.T) {
	approvals.VerifyArray(t, Answer(2))
}