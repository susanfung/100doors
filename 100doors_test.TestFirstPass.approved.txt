[0]=opened
[1]=opened
[2]=opened
[3]=opened
[4]=opened
[5]=opened
[6]=opened
[7]=opened
[8]=opened
[9]=opened
[10]=opened
[11]=opened
[12]=opened
[13]=opened
[14]=opened
[15]=opened
[16]=opened
[17]=opened
[18]=opened
[19]=opened
[20]=opened
[21]=opened
[22]=opened
[23]=opened
[24]=opened
[25]=opened
[26]=opened
[27]=opened
[28]=opened
[29]=opened
[30]=opened
[31]=opened
[32]=opened
[33]=opened
[34]=opened
[35]=opened
[36]=opened
[37]=opened
[38]=opened
[39]=opened
[40]=opened
[41]=opened
[42]=opened
[43]=opened
[44]=opened
[45]=opened
[46]=opened
[47]=opened
[48]=opened
[49]=opened
[50]=opened
[51]=opened
[52]=opened
[53]=opened
[54]=opened
[55]=opened
[56]=opened
[57]=opened
[58]=opened
[59]=opened
[60]=opened
[61]=opened
[62]=opened
[63]=opened
[64]=opened
[65]=opened
[66]=opened
[67]=opened
[68]=opened
[69]=opened
[70]=opened
[71]=opened
[72]=opened
[73]=opened
[74]=opened
[75]=opened
[76]=opened
[77]=opened
[78]=opened
[79]=opened
[80]=opened
[81]=opened
[82]=opened
[83]=opened
[84]=opened
[85]=opened
[86]=opened
[87]=opened
[88]=opened
[89]=opened
[90]=opened
[91]=opened
[92]=opened
[93]=opened
[94]=opened
[95]=opened
[96]=opened
[97]=opened
[98]=opened
[99]=opened